xsql/ast.go

```diff
@@ -1164,14 +1164,21 @@ func (v *ValuerEval) evalBinaryExpr(expr *BinaryExpr) interface{} {
 	case error:
 		return val
 	}
-
+	if isSliceOrArray(lhs) {
```

Comment:
The case of []interface{} can be combined with isSliceOrArray. Remove the case and leave isSliceOrArray only